python/tests/res/job_queue/test_job_queue_manager.py

```diff
@@ -24,16 +25,22 @@ def dummy_exit_callback(args):
     "exit_callback" : dummy_exit_callback
 }
 
+simple_script = "#!/usr/bin/env python\n"\
+                "with open('STATUS', 'w') as f:"\
+                "   f.write('finished successfully')"\
+                "\n"
 
-def create_queue():
+failing_script = "#!/usr/bin/env python\n"\
+                        "import sys\n"\
```

Comment:
Should we do something with the line alignment here?

Reply:
Perhaps this would be more readable?
```python
failing_script = """
#!/usr/bin/env python
import sys
sys.exit(1)
"""
```

Reply:
readability fixed